### PR TITLE
Update field names

### PR DESCRIPTION
**Pull request by aunroel**
Due to limitations on field names in OutSystems, we need to tweak some of the fields

### CODE REVIEW

File: data-shaping/output/output.py [hunk 1]
```diff
@@ -295,7 +295,7 @@ def transfer_exit_assessment_record(path, sheets):
             comments.append(row['OSCMNT'])
 
         ears = pd.DataFrame(
-            data={"id": ids, "training_record_id": training_ids, "date": dates, "type_id": types, "result_id": results, "comments": comments}
+            data={"id": ids, "training_record_id": training_ids, "date": dates, "type_id": types, "outcome_id": results, "comments": comments}
```
Comment (aunroel) [inline data-shaping/output/output.py:298]: for unfathomable to me reasons - using `result_id` makes OutSystems complain about circular FK dependency....

File: data-shaping/output/output.py [hunk 2]
```diff
@@ -642,26 +642,26 @@ def transfer_training_record_separate(path, id, hei_qualification_completed, hei
...
                 "hsst_arp_completion_date": hsst_arp_completion_date,
-                "hsst_d_clin_part_a_completed": hsst_d_clin_part_a_completed,
-                "hsst_d_clin_part_a_completion_date": hsst_d_clin_part_a_completion_date,
-                "hsst_d_clin_part_b_completed": hsst_d_clin_part_b_completed,
-                "hsst_d_clin_part_b_completion_date": hsst_d_clin_part_b_completion_date,
-                "hsst_d_clin_part_c1_completed": hsst_d_clin_part_c1_completed,
-                "hsst_d_clin_part_c1_completion_date": hsst_d_clin_part_c1_completion_date,
-                "hsst_d_clin_part_c2_completed": hsst_d_clin_part_c2_completed,
-                "hsst_d_clin_part_c2_completion_date": hsst_d_clin_part_c2_completion_date,
+                "hsst_dclin_part_a_completed": hsst_d_clin_part_a_completed,
```
Comment (William-Blackie) [inline data-shaping/output/output.py:657]: I'm happy to change these but we're mixing three variations of "completed", "comp" and "compl". Could we use just one?

Comment (aunroel) [reply data-shaping/output/output.py:657]: this is only due to length constrain...

if you prefer - I can change it all to `comp`?

**Review by William-Blackie — CHANGES_REQUESTED** [review]
Looks good, just a couple of inconsistencies with naming convention

File: data-shaping/output/output.py (hunk 2 above)
Comment (William-Blackie) [reply data-shaping/output/output.py:657]: If we have to bend due to to the length constraint then `comp` is fine

**Review by William-Blackie — APPROVED** [review]
Looks good!

**Pull request merged**